### ISSUE TITLE
It doesn't work for me on all worlds

### ISSUE THREAD

**Issue by Pipizio** [rcebulko/foundryvtt-ammo-tracker#3]
Hi, I'm having problems with this module: On some worlds it works for me, but on others it doesn't.
I am using as additional modules: Midi, DAE.

Does this module need some dependency to work?

**Comment by rcebulko** [comment]
It should not require any dependencies as far as I know. Are you using the DnD 5e system? Have you been using the "Create encounter" and "End combat" buttons?

**Comment by Pipizio** [comment]
Yes, I confirm that I have followed the procedure as per description, dnd5e and the fight.
I notice that on some worlds it works and others don't, so I think there is a compatibility problem with some modules.
I use a lot of them (108) so I find it really difficult to understand which one is causing the problem.

**Comment by rcebulko** [comment]
My best guess would be another module (possibly related to ammo or inventories) that's changing the item `consumableType` to something other than `ammo`, which is how this mod identifies ammunition items to track. There's a few ways I'd debug this:
1. Set breakpoints in the module script and inspect what items are getting picked up as ammo, and play from there (requires some JS comfort)
2. Clone your world (optional) and start a binary search for the "offending" modules. You have 108 modules (I'm pretty close to that myself), so disable the first 59 and reload. Does it work now? If so, the offender is in the first half; if not, it's in the second half. Now enable/disable half of the offending group, and so on. Each step cuts the list of possible offenders in half, meaning (if there's only one conflicting module) you can identify it within 7 iterations

**Comment by warteckel** [comment]
It does not work on my side neither. Maybe it is because I'm using a translated version (French). I'm using dnd5e and I start and end combat with the correct procedure.
